atlp-rwanda/falcons-ec-be
# #184637248 logout functionality

**What does this PR do?**

This PR adds a feature that allows user to logout once the api is invoked
**Description of Task to be completed?**

The logout feature enables users to safely sign out of the application or website, guaranteeing that their data is safeguarded against unauthorized access.

**How should this be manually tested?**

- clone the repository
- checkout ft-logout-functionality-184637248 branch
- Run npm install to install the dependencies and dev dependencies in your terminal
- Run npm run dev to start the server in terminal and check if server is working
- In URL field enter localhost:PORT/api-docs to open swagger UI
- Login users using /api/v1/users/signin endpoint
- Logout users using /api/v1/users/logout endpoint

**Any background context you want to provide?**
none
**What are the relevant pivotal tracker/Trello stories?**
tracker : #184637248
**Screenshots (if appropriate)**
none
**Questions:**
none
